docker-compose.yml

```diff
@@ -405,6 +405,7 @@ services:
         - LOG_LINES=${LOG_LINES:-9999}
         - ACME_CONTACT=${ACME_CONTACT:-}
         - ADDITIONAL_SAN=${ADDITIONAL_SAN}
+        - ACME_IGNORE_AUTOCONFS=${ACME_IGNORE_AUTOCONFS}
```

Comment:
Needs a default value (`${ACME_IGNORE_AUTOCONFS:-n}`) for people who don't run the update script.